specification/github-network/resource-manager/readme.md

```diff
@@ -59,42 +59,6 @@ input-file:
   - GitHub.Network/preview/2024-04-01-preview/GitHub.Network.json
 ```
 
-### Tag: package-2023-11-01-rc
```

Comment:
These versions are blocked by the service if traffic is sent to it, so no customer is using any of them.